.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: scala
 scala:
-  - 2.13.1
+  - 2.12.10
```

Comment:
Travis not a fan of 2.13?

Reply:
No, the AWS Lambda facades we're pulling in from IOLambda are only published for Scala 2.12 (so far… They would support 2.13 but the ScalablyTyped project doesn't have the resources to publish the artifacts yet 🙁)